src/Symfony/Bundle/FrameworkBundle/Kernel/MicroKernelTrait.php

```diff
@@ -48,7 +48,9 @@ trait MicroKernelTrait
      */
     private function configureContainer(ContainerConfigurator $container, LoaderInterface $loader, ContainerBuilder $builder): void
     {
-        $configDir = $this->getConfigDir();
+        if (!file_exists($configDir = $this->getConfigDir())) {
```

Comment:
The drawback of this approach is that if ppl make a typo in what getConfigDir returns, this will be silently ignored. 
A better approach might be to make getConfigDir return `$this->getProjectDir().'/{config}';` instead

Reply:
Nice idea, it works well `->import()` but it doesn't for `getBundlesPath()` which is `getConfigDir()` dependant

Reply:
I did a workaround in `getBundlesPath()` but I'm not too happy with it... yceruto